### DISCUSSION TITLE
Is there a message queue available in Rocket?

### DISCUSSION THREAD

**Discussion by swdcgghxa** [rwf2/Rocket#1781] [Questions]
Rocket中是否有可用的消息队列？
Is there a message queue available in Rocket?

比如python中有Celery。
For example, there is Celery in python.
[Celery message queue](https://docs.celeryproject.org/en/stable/index.html)

比如starlette里面有后台任务。
For example, there are background tasks in starlette. 
[starlette](https://www.starlette.io/background/)

請問我如何在Rocket中使用MQ或有什麼標準作法。
How do I use MQ in Rocket or what is the standard practice?

😀😀😀

**Comment by SergioBenitez** [comment]
This is a duplicate of https://github.com/SergioBenitez/Rocket/discussions/1735.
